released/packages/coq-coqeal/coq-coqeal.1.0.5/opam

```diff
@@ -0,0 +1,49 @@
+
+opam-version: "2.0"
+maintainer: "Cyril Cohen <cyril.cohen@inria.fr>"
+
+homepage: "https://github.com/CoqEAL/coqeal"
+dev-repo: "git+https://github.com/CoqEAL/coqeal.git"
+bug-reports: "https://github.com/CoqEAL/coqeal/issues"
+license: "MIT"
+
+synopsis: "CoqEAL - The Coq Effective Algebra Library"
+description: """
+This libary contains a subset of the work that was developed in the context of the ForMath european project (2009-2013). It has two parts:
+- theory (module CoqEAL_theory), which contains formal developments in algebra and optimized algorithms on mathcomp data structures.
+- refinements (module CoqEAL_refinements), which is a framework to ease change of data representation during a proof."""
+
+build: [make "-j%{jobs}%" ]
+install: [make "install"]
+depends: [
+  "coq" {(>= "8.10" & < "8.14~") | (= "dev")}
+  "coq-bignums" 
+  "coq-paramcoq" {(>= "1.1.1") | (= "dev")}
+  "coq-mathcomp-multinomials" {((>= "1.5.1" & < "1.7~") | = "dev")}
+  "coq-mathcomp-algebra" {((>= "1.11.0" & < "1.13~") | = "dev")}
+]
+
+tags: [
+  "category:Computer Science/Decision Procedures and Certified Algorithms/Correctness proofs of algorithms"
+  "keyword:effective algebra"
+  "keyword:elementary divisor rings"
+  "keyword:Smith normal form"
+  "keyword:mathematical components"
+  "keyword:Bareiss"
+  "keyword:Karatsuba"
```

Comment:
Please use "`Karatsuba multiplication`", so as to not create a duplicate keyword. (See https://coq.inria.fr/opam/www/).

Reply:
My bad, well catch , thanks!